File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_cs_samples/App_Code/DataBinding/Employee.cs
namespace DataBinding {
    public class Employee {
        private string id;
        private string department;
        private string name;
        private double salary;

        public Employee(string id, string name, string department, double salary) {
            this.id = id;
            this.name = name;
            this.department = department;
            this.salary = salary;
        }
        public Employee() {
        }
        public string ID {
            get { return id; }
            set { id = value; }
        }
        public string Name {
            get { return name; }
            set { name = value; }
        }
        public string Department {
            get { return department; }
            set { department = value; }
        }
        public double Salary {
            get { return salary; }
            set { salary = value; }
        }
    }
}

File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_vb_samples/App_Code/VisualBasicWebApp/Employee.vb
Imports Microsoft.VisualBasic
Namespace VisualBasicWebApp
    Public Class Employee
        Private _id As String
        Private _department As String
        Private _name As String
        Private _salary As Double
        Public Sub New()
        End Sub
        Public Sub New(ByVal id As String, ByVal name As String, ByVal department As String, ByVal salary As Double)
            _id = id
            _name = name
            _department = department
            _salary = salary
        End Sub
        Public Property ID() As String
            Get
                Return _id
            End Get
            Set(ByVal value As String)
                _id = value
            End Set
        End Property
        Public Property Name() As String
            Get
                Return _name
            End Get
            Set(ByVal value As String)
                _name = value
            End Set
        End Property
        Public Property Department() As String
            Get
                Return _department
            End Get
            Set(ByVal value As String)
                _department = value
            End Set
        End Property
        Public Property Salary() As Double
            Get
                Return _salary
            End Get
            Set(ByVal value As Double)
                _salary = value
            End Set
        End Property
    End Class
End Namespace

